README.md

```diff
@@ -28,13 +28,11 @@
 
 ## Why driftctl ?
 
-Infrastructure as code is awesome, but there are too many moving parts: codebase, state file, actual cloud state. Things tend to drift.
+Infrastructure drift is a blind spot and a source of potential security issues.
+Drift can have multiple causes: from team members creating or updating infrastructure through the web console without backporting changes to Terraform, to unexpected actions from authenticated apps and services.
 
-Drift can have multiple causes: from developers creating or updating infrastructure through the web console without telling anyone, to uncontrolled updates on the cloud provider side. Handling infrastructure drift vs the codebase can be challenging.
+Spot discrepancies as they happen : driftctl is a free and open-source CLI that warns of infrastructure drift and fills in the missing piece in your DevSecOps toolbox.
 
-You can't efficiently improve what you don't track. We track coverage for unit tests, why not infrastructure as code coverage?
```

Comment:
Why remove this sentence ? I think it's still something interesting to talk about coverage.

Reply:
okay let's keep it. Just tried to keep it short and simple, but this sentence is still valid obviously.